tests/unit/db_updates_tests.py

```diff
@@ -34,25 +34,36 @@ class DbUpdatesTestsBase(UnitTestDbBase):
     """
     Common _db_updates tests methods
     """
-
+    
     def setUp(self):
         """
         Set up test attributes
         """
         super(DbUpdatesTestsBase, self).setUp()
         self.client.connect()
-        self.new_dbs = list()
+        self.create_db_updates()
+        self.create_dbs()
 
     def tearDown(self):
         """
         Reset test attributes
         """
-        [db.delete() for db in self.new_dbs]
+        self.delete_db_updates()
         self.client.disconnect()
         super(DbUpdatesTestsBase, self).tearDown()
 
-    def create_dbs(self, count=3):
+    def create_dbs(self, count=2):
         self.new_dbs += [(self.client.create_database(self.dbname())) for x in range(count)]
+        # Verify that all created databases are listed in _db_updates
+        all_dbs_exist = False
+        while not all_dbs_exist:
+            changes = list()
+            feed = Feed(self.client, timeout=1000)
+            for change in feed:
+                changes.append(change)
+                if len(changes) == 3:
```

Comment:
Does the count of 3 include the _users db?

Reply:
Actually, it looks like this assert doesn't account for the case where `count` is set to the non-default value.

Reply:
@tomblench The _users and _replicator dbs are created before _global_changes.  During the creation of _global_changes in the setup of `db_updates_tests.py`, the only databases that should be seen in the feed are `_dbs` (included during `_global_changes` creation) and the two databases for testing in `create_dbs`.

You're right that the assertion only works for case `count=2`.  Since the tests only need two databases, I will remove the `count` parameter.